src/Tribe/Template/Month.php

```diff
@@ -619,8 +619,12 @@ private function get_daily_events( $date ) {
 					'update_post_term_cache' => false,
 					'update_post_meta_cache' => false,
 					'no_found_rows'          => false,
+					'eventDoNotInjectDate'   => true,
```

Comment:
CamelCase seems a bit jarring next to all the snake cased field names ... are we sure we don't want to go for consistency here?